code/modules/clothing/head/misc.dm

```diff
@@ -267,26 +243,12 @@
 	flags_inv = BLOCKHAIR
 	body_parts_covered = 0
 
-/obj/item/clothing/head/hijab
```

Comment:
whats up with this? I dont think islam is entirely gone, but there are non religious reasons to wear it too